website/docs/d/actions_environment_secrets.html.markdown

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
@tjcorr do you mind linking to these excellent new docs in the `github.erb` file?